hermes/internal/notification/payloads/messagepayloads.go

```diff
@@ -31,15 +31,18 @@ type Request struct {
 }
 
 type PayloadRequest struct {
-	ExternalId uuid.UUID       `json:"externalId"`
+	ExternalId uuid.UUID       `json:"externalId"` //workspace-id
```

Comment:
```suggestion
	ExternalId uuid.UUID       `json:"externalId"`
```

Suggestion: avoid comments